app/controllers/application_controller.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::API
+        include DeviseTokenAuth::Concerns::SetUserByToken
+  before_action :authenticate_user!
```

Comment:
```suggestion
  include DeviseTokenAuth::Concerns::SetUserByToken
  before_action :authenticate_user!
```

Let's made this well formated :)

Reply:
On it!